c10/core/TensorImpl.h

```diff
@@ -1344,6 +1347,35 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
     device_opt_ = storage_.device();
   }
 
+  virtual void empty_tensor_restride(MemoryFormat memory_format) {
```

Comment:
There really ought to be a comment on this function.

Reply:
added